src/Cron/DayOfMonthField.php

```diff
@@ -70,6 +70,21 @@ public function isSatisfiedBy(DateTime $date, $value)
             return $fieldValue == $date->format('t');
         }
 
+        // Find out if this is the last workday of the month
+        if ( $value == 'D' ) {
```

Comment:
you should state somewhere that workday is defined as monday-friday (in some businesses workday is defined differently)